### PR TITLE
[Resources] Update Resources.ManagementGroups Module

### PR DESCRIPTION
**Pull request by git-thomasdolan**
## Description

- Updated the ManagementGroups SDK to the latest version
- Added and updated tests to test the new changes

## Checklist

- [X] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by git-thomasdolan** [comment]
@dingmeng-xue Can someone review this? Thanks!

File: src/Resources/Resources/Az.Resources.psd1 [hunk 1]
```diff
@@ -181,7 +180,12 @@ CmdletsToExport = 'Get-AzProviderOperation', 'Remove-AzRoleAssignment',
                'Remove-AzResourceManagementPrivateLink', 
                'New-AzResourceManagementPrivateLink', 
                'New-AzPrivateLinkAssociation', 'Get-AzPrivateLinkAssociation', 
-               'Remove-AzPrivateLinkAssociation'
+               'Remove-AzPrivateLinkAssociation',
+               'Get-AzEntity', 'Get-AzManagementGroupNameAvailability',
+               'Get-AzTenantBackfillStatus', 'Start-AzTenantBackfill',
+	           'Get-AzManagementGroupSubscription', 'Get-AzSubscriptionUnderManagementGroup',
+	           'New-AzHierarchySetting', 'Update-AzHierarchySetting',
+               'Get-AzHierarchySetting', 'Remove-AzHierarchySetting'
```
Comment (isra-fel) [inline src/Resources/Resources/Az.Resources.psd1:188]: Inconsistent indentation

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @git-thomasdolan for a PR that adds new cmdlets, a cmdlet design review has to be approved first. So please submit an issue in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) and fill in the questionare.

Also in this PR you need to update the ChangeLog.md under src/Resources.

**Comment by git-thomasdolan** [comment]
Created issue [here](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1180)

**Comment by git-thomasdolan** [comment]
Hi,

Are there any other changes I need to make? I believe I made all the requested changes from earlier. Thanks!

**Review by wyunchi-ms — APPROVED** [review]

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**